build.py

```diff
@@ -32,7 +32,10 @@ def run(cmd: str, **kwargs) -> subprocess.CompletedProcess:
 
 def build():
     run(f"{PYTHON_EXECUTABLE_PATH} --version")
-    run(f"{PYTHON_EXECUTABLE_PATH} misc/generate_pyqt.py")
+
+    if not os.environ.get("SKIP_GENERATE_PYQT", False):
```

Comment:
I'd rather keep thing simple and just remove `/venv/lib/python3.9/site-packages/parsec/core/gui/_resources_rc.py` as part of the Dockerfile

(we may want a better way of doing this later on if we are required to build a docker image for production, but just stay KISS for the moment)

Reply:
That problem I was getting is I can't build pyqt on alpine even when installing `py3-qt5`.

Skipping `generate_pyqt` allow to not install `pyqt` deps